comtypes/test/test_enum.py

```diff
@@ -0,0 +1,26 @@
+import contextlib
+import unittest as ut
+
+from comtypes import POINTER
+import comtypes.client
+
+
+class Test_Enum(ut.TestCase):
```

Comment:
As I suggested in #345, I plan to introduce the [`enum`](https://docs.python.org/3/library/enum.html) module.

There are also `CoClass`es those name starts with `Enum`.

To emphasize the difference from these, I think the names should be `class Test_IEnum`, `def test_ienum`, and `test_ienum.py`.